biosimulators_utils/gh_action/core.py

```diff
@@ -7,6 +7,7 @@
 """
 
 from .data_model import Comment, GitHubActionCaughtError
+from ..globals import JSONType
```

Comment:
two dots? was this intended?

Reply:
yes, `globals` is in `biosimulator_utils`, not `biosimulator_utils/gh_action`

Reply:
ok, I see it again below, so this must be intended. Didn't realize it was using relative imports.